### DISCUSSION TITLE
Updated MRtrix (non-release version)

### DISCUSSION THREAD

**Discussion by timrosenow** [neurodesk/neurodesk.github.io-hugo#426] [New Tool]
Hi there.

In August, the MRtrix team fixed an important bug relating to opening certain DICOM files that come from XNAT (details here if you are interested: https://github.com/MRtrix3/mrtrix3/pull/2691 ) . This has been fixed in the current stable version, but is not yet part of an official release.

If it's not too much trouble, would it be possible to include a version of mrtrix on neurodesktop that contains this fix, i.e. perhaps the most recent stable version built from source? I am happy to help with whatever is needed to do so.

**Comment by stebo85** [comment]
Dear @timrosenow 

that should be possible as we are building mrtrix from source already. Can you propose the change here through a pull request: https://github.com/NeuroDesk/neurocontainers/blob/master/recipes/mrtrix3/build.sh

thank you
Steffen

**Comment by stebo85** [comment]
this is completed now. Will remove this build again when a new mrtrix release is out :)
